### PR TITLE
DAT-3990 Introduce infobox to ThemeDesigner

### PR DESCRIPTION
**Pull request by sebastianmarzjan**
## LINKS

https://wikia-inc.atlassian.net/browse/DAT-3990
## DESCRIPTION

This pull request introduces an infobox in place of image inside of a Theme Designer preview window. There was purposely no rework of Theme Designer's structure done as part of that change given the plans to address theming in a holistic manner this year.

### CODE REVIEW

File: extensions/wikia/ThemeDesigner/templates/ThemeDesigner_Preview.php [hunk 1]
```diff
@@ -1,4 +1,23 @@
-<?= ThemeDesignerHelper::parseText("[[File:Aquarium.jpg|thumb|Quisque pellentesque vestibulum ullamcorper.]]
+<?= ThemeDesignerHelper::parseText("
+<infobox <?= $infoboxTheme ?>>
```
Comment (idradm) [inline extensions/wikia/ThemeDesigner/templates/ThemeDesigner_Preview.php:2]: Can we just use `theme="europa"` here?

**Comment by sebastianmarzjan** [comment]
Test failure not related, filing a ticket

File: extensions/wikia/ThemeDesigner/templates/ThemeDesigner_Preview.php (hunk 1 above)
Comment (sebastianmarzjan) [inline extensions/wikia/ThemeDesigner/templates/ThemeDesigner_Preview.php:2]: Not until the theme is sitewide

File: extensions/wikia/ThemeDesigner/ThemeDesignerController.class.php [hunk 2]
```diff
@@ -89,7 +89,9 @@ public function executePicker() {
 	}
 
 	public function executePreview() {
+		global $wgEnablePortableInfoboxEuropaTheme;
 
+		$this->infoboxTheme = $wgEnablePortableInfoboxEuropaTheme ? 'theme="europa"' : '';
```
Comment (idradm) [inline extensions/wikia/ThemeDesigner/ThemeDesignerController.class.php:94]: If you set `theme="europa"` in infobox markup this part won't be needed. What do you think about that?

**Comment by idradm** [comment]
LG, comments about logic switch are optional

**Comment by sebastianmarzjan** [comment]
The test failure is related to https://wikia-inc.atlassian.net/browse/MAIN-6740 (integration test, unrelated to this PR)

**Pull request merged**